ts/services/addGlobalKeyboardShortcuts.ts

```diff
@@ -75,22 +72,7 @@ export function addGlobalKeyboardShortcuts(): void {
       // elements match (generally going to be a parent element)
       const focusedIndex = focusedIndexes.at(-1) ?? -1;
 
-      const lastIndex = targets.length - 1;
-      const increment = shiftKey ? -1 : 1;
-
-      let index;
-      if (focusedIndex < 0 || focusedIndex >= lastIndex) {
-        index = 0;
-      } else {
-        index = focusedIndex + increment;
-      }
-
-      while (!targets[index]) {
-        index += increment;
-        if (index > lastIndex || index < 0) {
-          index = 0;
-        }
-      }
+      const index = (focusedIndex + 1) % targets.length;
```

Comment:
I removed `shiftKey` handling because it's disabled for Ctrl+T as [Ctrl+Shift+T is the shortcut for focusing the composer field](https://github.com/signalapp/Signal-Desktop/blob/19083cadf7136b62616132c2698ba993d69a3622/ts/services/addGlobalKeyboardShortcuts.ts#L263-L265). Ctrl+Shift+F6 would still be possible if you think users would suddenly start caring about that shortcut, since it's been broken for a long time without anyone noticing.

```suggestion
      const lastIndex = targets.length - 1;

      let index;
      if (focusedIndex < 0) {
        index = shiftKey ? lastIndex : 0;
      } else {
        const increment = shiftKey ? lastIndex : 1;
        index = (focusedIndex + increment) % targets.length;
      }
```